### ISSUE TITLE
Can't Select Multiple Instrument Parts to Open

### ISSUE THREAD

**Issue by joecorporon** [musescore/MuseScore#18456]
### Issue type

Other type of issue

### Bug description

When writing a score for multiple instruments I should be able to open several at once from the Parts dialog box. However I can only open one at a time or all of them. The ability to select multiple parts doesn't work.

From the Musescore 4 Handbook:
"You can also select specific parts to open at once. Do this by holding Control (Mac: ⌘) while selecting the parts you’d like to open, then click Open selected. You can also select a range of contiguous parts by clicking the first and holding Shift while clicking the last."

Neither option for selecting multiple parts works. It's either one, or all.

### Steps to reproduce

1. Open the Parts dialog.
2. Attempt to select multiple parts either by pressing Control, or several contiguous by selecting first part and Shift indicating the last part.
3. Neither way to select multiple parts works. I can only open one part at a time, or all the parts at once.

### MuseScore Version

OS: Manjaro Linux, Arch.: x86_64, MuseScore version (64-bit): 4.0.2-, revision: github-musescore-musescore-mus

### Regression

No.

### Operating system

Manjaro Linux 23.0.0

**Comment by bkunda** [comment]
Unable to reproduce on macOS and Win.
@zacjansheski can you please try this in Linux to see if it's an OS-specific problem? 
The only other thought I had @joecorporon was whether your keyboard/language could perhaps be causing issues with the Ctrl and Shift keys 🤷‍♂️

**Comment by joecorporon** [comment]
I can't rule that out except to say the Ctrl and Shift keys work fine in every other way within Musescore and all the other software I have installed. My system language is US English.

**Comment by zacjansheski** [comment]
I am unable to reproduce this issue in Ubuntu 22.04.2 in MS4.0.2 or the Nightly.

Please check if this is score specific or if it occurs with all of your scores.
It might help to try this on the nightly and/or doing a "Revert to Factory Settings"

**Comment by MarcSabatella** [comment]
It's definitely an issue for me on Linux (Debian 11, running as a container within ChromeOS), both in 4.0.2 and 4.1.  To be clear, this is about multi-selecting parts in the Psrts window, not the Instruments panel where it works as expected.

I think it might have worked at one time - see https://github.com/musescore/MuseScore/issues/11848 where I seem to be implying I *can* multiselect with mouse, just not with the keyboard.

Here is a screen recording showing me try in vain to multiselect via Shift+click, Ctrl+click, and Shift+Up/Down:

![multiselect](https://github.com/musescore/MuseScore/assets/1799936/0b07dbd1-4c92-49d4-b9d3-ce87bb6077a8)

**Comment by zacjansheski** [comment]
Understood, it is working as expected on my device

https://github.com/musescore/MuseScore/assets/69917893/f8b0b878-bea1-401c-8125-5a79b051e687

**Comment by oktophonie** [comment]
Also cannot reproduce in 4.0.2 or 4.1 (Artix); curious

**Comment by joecorporon** [comment]
My issue seems to duplicate the issue that @MarcSabatella is having on linux. Cannot multi-select parts in the Parts windows with keyboard or mouse/keyboard.

**Comment by MarcSabatella** [comment]
Interesting!  I'll guess it relates to desktop environment or some other variable like that.  Mine is ChromeOS so I expect occasional weirdness (but this is somehow especially so), but I am also setting things up to force some GTK behaviors and other tweaks I've come up with over the years to make things run smoothly.  I'll play around some later to see if I can figure out the trigger.

**Comment by ghost** [comment]
(Combined into https://github.com/musescore/MuseScore/issues/11848)
